nextflow_schema.json

```diff
@@ -8,86 +8,293 @@
         "input_output_options": {
             "title": "Input/output options",
             "type": "object",
+            "fa_icon": "fas fa-terminal",
+            "description": "Define where the pipeline should find input data and save output data.",
+            "required": [
+                "input"
+            ],
             "properties": {
                 "input": {
                     "type": "string",
                     "fa_icon": "fas fa-dna",
-                    "description": "Input FastQ files.",
-                    "help_text": "A glob pattern for input FastQ files. Should include at least one asterisk (*). For paired-end data, should contain curly brackets with two patterns differentiating the paired reads e.g. `*_R{1,2}.fastq.gz`"
-                },
-                "single_end": {
-                    "type": "boolean",
-                    "description": "Specifies that the input is single-end reads.",
-                    "fa_icon": "fas fa-align-center",
-                    "default": false,
-                    "help_text": "By default, the pipeline expects paired-end data. If you have single-end data, specify this parameter on the command line when you launch the pipeline. It is not possible to run a mixture of single-end and paired-end files in one run."
+                    "description": "Input DIA mzML files",
+                    "help_text": "Use this to specify the location of your input DIA mzML files. For example:\n\n```bash\n--input ''path/to/data/sample_*.mzML'\n```\n\nPlease note the following requirements:\n\n1. The path must be enclosed in quotes\n2. The path must have at least one `*` wildcard character\n\nIf left unspecified, a default pattern is used:  'data/sample_*.mzML'",
+                    "default": "data/sample_*.mzML"
                 },
                 "outdir": {
                     "type": "string",
                     "description": "The output directory where the results will be saved.",
-                    "default": "./results",
+                    "default": ".&#x2F;results",
```

Comment:
This weird escape parameters are also showing up here `&#x2F;`, I've asked already if that is normal

Reply:
is there a consensus now how to deal with this?

Reply:
yes, the `/` should be kept